pipelines/internal/commands/run/run.go

```diff
@@ -365,6 +366,16 @@ func buildRequest(filename, project string) (*genomics.RunPipelineRequest, error
 	}
 
 	pipeline.Actions = []*genomics.Action{mkdir(directories)}
+
+	if *debug {
+		debugAction := &genomics.Action{
+			ImageUri:     "anamanolache/sshserver:1.0",
```

Comment:
for now, this should be the same as how I construct the fuse image (eg, the project's GCR repository)

eventually we'll host a copy at gcr.io/cloud-genomics-pipelines or similar